awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -1,35 +1,156 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Link, withRouter } from 'react-router-dom';
 import { withI18n } from '@lingui/react';
 import { t } from '@lingui/macro';
 import { CardBody, Button } from '@patternfly/react-core';
 import styled from 'styled-components';
+import { DetailList, Detail } from '@components/DetailList';
+import { ChipGroup, Chip, CredentialChip } from '@components/Chip';
+import { VariablesInput } from '@components/CodeMirrorInput';
+import { toTitleCase } from '@util/strings';
+import { Job } from '../../../types';
 
 const ActionButtonWrapper = styled.div`
   display: flex;
   justify-content: flex-end;
 `;
-class JobDetail extends Component {
-  render() {
-    const { job, i18n } = this.props;
 
-    return (
-      <CardBody>
-        <b>{job.name}</b>
+const VERBOSITY = {
+  0: '0 (Normal)',
+  1: '1 (Verbose)',
+  2: '2 (More Verbose)',
+  3: '3 (Debug)',
+  4: '4 (Connection Debug)',
+};
 
-        <ActionButtonWrapper>
-          <Button
-            variant="secondary"
-            aria-label="close"
-            component={Link}
-            to="/jobs"
-          >
-            {i18n._(t`Close`)}
-          </Button>
-        </ActionButtonWrapper>
-      </CardBody>
-    );
-  }
+function JobDetail({ job, i18n }) {
```

Comment:
I went back and forth a lot about whether to make a different component for each type of job. In the end it just felt like way too much repeated code with very little variation. We can always split them back out down the line if the different types end up deviating further from each other.